src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp

```diff
@@ -238,7 +238,7 @@ void AieTracePluginUnified::updateAIEDevice(void *handle) {
 
   // First, check against memory bank size
   // NOTE: Check first buffer for PLIO; assume bank 0 for GMIO
-  uint8_t memIndex = isPLIO ? deviceIntf->getAIETs2mmMemIndex(0) : 0;
+  uint8_t memIndex = (isPLIO && (deviceIntf != nullptr))? deviceIntf->getAIETs2mmMemIndex(0) : 0;
```

Comment:
This is difficult to debug with a break point.  I recommend ternaries that call functions to be split as
```
(cond) 
? true_path()
: false_path();

```

Reply:
Let me fix this.